client/blocks/post-edit-button/style.scss

```diff
@@ -31,6 +31,33 @@
 			display: none;
 		}
 	}
+
+	&.tooltip {
```

Comment:
Should we consolidate this with fbhepr%2Skers%2Spnylcfb%2Spyvrag%2Soybpxf%2Spbzzrag%2Qohggba%2Sfglyr.fpff%3Se%3Q6on57o46%2324-og so that we don't have two copies of the same thing?

Perhaps in fbhepr%2Skers%2Spnylcfb%2Spyvrag%2Sernqre%2Sfglyr.fpff%3Se%3Q064or035-og so it's accessible anywhere in the reader?

Reply:
I like the idea. I opted to create new mixins, to keep the components clearly separate from each other. 04226561b298fbfaeb7432c46650d9b8fc63a5d7


This now impacts other places where we were using that tooltip:

<img width="371" alt="Screenshot 2025-07-03 at 17 26 35" src="https://github.com/user-attachments/assets/6b2356bd-224b-4cd3-a859-7244a7ba496f" />
<img width="232" alt="Screenshot 2025-07-03 at 17 26 38" src="https://github.com/user-attachments/assets/969c4692-9fb8-417e-afb8-8c082fd80221" />